com.unity.ml-agents/Runtime/Inference/BarracudaModelParamLoader.cs

```diff
@@ -121,7 +122,7 @@ public static FailedCheck Error(string message)
                     CheckInputTensorShapeLegacy(model, brainParameters, sensors, observableAttributeTotalSize)
                 );
             }
-            else if (modelApiVersion == (int)ModelApiVersion.MLAgents2_0)
+            else if (modelApiVersion == (int)ModelApiVersion.MLAgents2_0 || modelApiVersion == (int)ModelApiVersion.MLAgents2_0_Recurrent)
```

Comment:
`>= ModelApiVersion.MLAgents2_0`? That will future-proof it (even if you don't combine MLAgents2_0 and MLAgents2_0_Recurrent)